Realm/RealmSwift/List.swift

```diff
@@ -72,28 +128,57 @@ public final class List<T: Object>: ListBase, SequenceType {
         }
     }
 
+    /// Returns the first object in the list, or `nil` if empty.
     public var first: T? { return _rlmArray.firstObject() as T? }
 
+    /// Returns the last object in the list, or `nil` if empty.
     public var last: T? { return _rlmArray.lastObject() as T? }
 
-    // MARK: Subarray Retrieval
+    // MARK: Filtering
+
+    /**
+    Returns `Results` containing list elements that match the given predicate.
+
+    :param: predicateFormat The predicate format string which can accept variable arguments.
```

Comment:
How about "The predicate format string, optionally followed by a variable number of arguments."?

Or "The predicate format string and optional format arguments which will be passed to `NSPredicate(format:arguments:)`."?